core/base/Dockerfile

```diff
@@ -12,7 +12,19 @@ ARG MAILU_GID=1000
 RUN set -euxo pipefail \
   ; addgroup -Sg ${MAILU_GID} mailu \
   ; adduser -Sg ${MAILU_UID} -G mailu -h /app -g "mailu app" -s /bin/bash mailu \
-  ; apk add --no-cache bash ca-certificates curl python3 tzdata
+  ; apk add --no-cache bash ca-certificates curl python3 tzdata \
+  ; machine="$(uname -m)" \
+  ; if [[ "${machine}" == x86_64 || "${machine}" == armv8* || "${machine}" == aarch64 ]] \
+  ; then \
+      echo "Installing hardened-malloc" \
+  ;   apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing hardened-malloc \
+  ; else \
+      echo "Faking the install of hardened-malloc" \
+  ;   cp /lib/ld-musl-${machine}.so.1 /usr/lib/libhardened_malloc.so \
+  ;   touch /usr/lib/libhardened_malloc.so.faked \
```

Comment:
I think you can set LD_PRELOAD to /usr/lib/libhardened_malloc.so without any side-effect if it's not present.
Faking is not necessary and will make people using ldd only think it would be enabled when its not.

Reply:
Fair point. I know that other linkers complain when this is invalid... I haven't tested with musl.

Done in 81628149a234985e1fba07881ac3448078ce2bc3